assets/data/boss/weekly-boss/the-knave/en.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "The Knave",
+  "description": "The Knave, Fourth of the Fatui Harbingers.\nThe ancient balemoon blood flows within her fate, granting her all that the dynasty once had — yearning, power, and curses alike.\nIn her short life, she has supped of countless agonies and drunk deep of the cup of suffering, just as her ancestors once did when they imbibed poison at the rising of the dark sun.\nStill, neither suffering nor fate has proved a fatal toxin to her.\nInstead, they are the firewood fueling her quietly burning hearth, and are the fire bringing her children warmth.\nSomeday, the hearth-fire's faint radiance shall burn the old world away, incinerating the final scion of the baleful moon as well.\nSomeday, these still immature children will break free of ancient fate and usher in the future without tears she dreams of.\nBut until that time comes, until dawn breaks, the one named Peruere shall stand guard over their fragile, sweet dreams.\n\"If fate should pave for us a pale road, let us dye it the bloody crimson of the bale moon.\"\n\"Even a flickering, famished fire can burn all suffering away before daybreak.\"",
```

Comment:
this description should be updated to use the in game descript as opposed to the wiki description.
See [this pr](https://github.com/genshindev/api/pull/273/files) for reference